src/modules/Dashboard/components/ParticipantSnippet/ParticipantSnippet.tsx

```diff
@@ -1,18 +1,15 @@
-import { ElementType } from 'react';
-import { BoxTypeMap } from '@mui/system';
-
 import { Chip } from 'shared/components';
 import { StyledBodyLarger, StyledFlexTopCenter, variables } from 'shared/styles';
 
 import { ParticipantSnippetProps } from './ParticipantSnippet.types';
 
-export const ParticipantSnippet = <T extends ElementType = BoxTypeMap['defaultComponent']>({
+export const ParticipantSnippet = ({
```

Comment:
We have to keep the type parameter here, otherwise typescript complains about this props spread

https://github.com/ChildMindInstitute/mindlogger-admin/blob/7460397c388471a5692244087f997ce2094612c2/src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx#L83-L89

I thought I'd be able to get around this by specifying the `component` prop, but the typing information doesn't seem to propagate to the `ParticipantSnippetProps`, which maintains its default value

Reply:
Yup, I can see why this would fail.

What do you think about just making do with `ParticipantSnippet` always rendering a `div`, and wrapping yours in an `<li>`, just to keep things a little simpler? Otherwise I think we'll have to propagate that awkward generic syntax in several contexts… maybe it's not as bad as I think, but it seemed to be having a bit of a domino effect when I was trying to resolve it earlier.

Reply:
I can live with wrapping it in a `<li>`, but I think the generic syntax is only necessary if you're trying to spread props that you don't control. Like in this instance, the Autocomplete is expecting a list item element. Do you foresee this applying anywhere else?

Reply:
I solved it by doing a minor refactor, so I preserved the generic syntax and it shouldn't affect your code. 🙂